projects/rabbitmq.com/package.yml

```diff
@@ -0,0 +1,52 @@
+distributable:
+  url: https://github.com/rabbitmq/rabbitmq-server/releases/download/v{{version}}/rabbitmq-server-generic-unix-{{version}}.tar.xz
+  strip-components: 1
+versions:
+  github: rabbitmq/rabbitmq-server
+warnings:
+  - vendored
+dependencies:
+  erlang.org: '*'
+build:
+  dependencies:
+    python.org: ~3.11
```

Comment:
What is using python?

Reply:
I looked up this dependency here: https://github.com/Homebrew/homebrew-core/blob/e0e87ed974ed800f73dfe7ebcc49af030b088a16/Formula/r/rabbitmq.rb 😇